### DISCUSSION TITLE
Export Multiple PDFs as Separate (Individual) Files

### DISCUSSION THREAD

**Discussion by Erielson1708** [l1xnan/obsidian-better-export-pdf#240] [Ideas]
Currently, there is already the option to export multiple PDFs as a single file. Which is already great. But wouldn't it be possible to export each note as a different PDF? Something similar to an automation that saves a lot of work, selecting several notes and creating a PDF for each one, separately. Thank you for your attention.

**Comment by l1xnan** [comment]
Thanks for your feedback, it is on my to-do list. In fact, I have completed half of the work, but I have been a little busy in the last month and have no time to complete it further. It may need to wait for another half month.

**Reply by l1xnan** [reply]
@Erielson1708 I in the 1.9.0 version of the initial support, you can batch export all the notes under the folder as a PDF file, the use of the process of any problem, welcome to feedback!

**Reply by Erielson1708** [reply]
Thanks. This saved me a lot. And congratulations on the plugin, so useful and versatile.
